Marlin/src/lcd/menu/menu_advanced.cpp

```diff
@@ -443,7 +433,10 @@ void menu_backlash();
     #if ENABLED(DISTINCT_E_FACTORS)
       EDIT_ITEM_FAST(long5_25, MSG_AMAX_E, &planner.settings.max_acceleration_mm_per_s2[E_AXIS_N(active_extruder)], 100, max_accel_edit_scaled.e, []{ planner.reset_acceleration_rates(); });
       LOOP_L_N(n, E_STEPPERS)
-        EDIT_ITEM_FAST_N(long5_25, n, MSG_AMAX_EN, &planner.settings.max_acceleration_mm_per_s2[E_AXIS_N(n)], 100, max_accel_edit_scaled.e, []{ _reset_e_acceleration_rate(MenuItemBase::itemIndex); });
+        EDIT_ITEM_FAST_N(long5_25, n, MSG_AMAX_EN, &planner.settings.max_acceleration_mm_per_s2[E_AXIS_N(n)], 100, max_accel_edit_scaled.e, []{
```

Comment:
why this was changed?

Reply:
Hello.
I do not know. They are not related to this bug.
At the time of uploading my push_request, the comparison took an older version of 'menu_advanced.cpp'.
I don't know what the problem was, because my branch was synchronized with bugfix_2.0.x, where these modifications were already merged.

Reply:
So, do you know how to rever?